### ISSUE TITLE
Feature Request: Ability to add content

### ISSUE THREAD

**Issue by peterjem** [kylephillips/wp-nested-pages#308]
Thanks for this wonderful plugin. A suggestion, it would have been pretty much cool if there was an option to add content in the post(s) creation form, it would further help in making the post(s) creation process easier, simple and quick.

Thanks

**Comment by Sandstedt** [comment]
Yeah the "Add child" button with the current ability to create multiple child posts at the same time is golden! But would be even better if you could, ex choose to display some essential ACF fields there too. But maybe out of scope of this plugin (this plugin is probably a better fit for that: https://wordpress.org/plugins/acf-quickedit-fields/). Could help front-end wise if you need help, but PHP is not my best expertise.
